randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1603,78 +1411,34 @@ Extra - asset_id: collision_camera_018
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Dock to Big Hub
-      Morph Ball and After Dairon - Central Unit
+      Any of the following:
+          Flash Shift or Flash Shift Skip (Beginner) or Lay Cross Bomb or Use Spin Boost
+          Morph Ball and After Dairon - Central Unit
   > Dock to EMMI Zone Exit North
       Speed Booster and Open Charge Door
```

Comment:
This would need a Not Door Lock Randomizer on it too, or instead of Open Charge Door, the event for the speed blocks before ferenia. That should cover coming from this door and if you came from below (cause you can just use the speed from down there then).